api/migrations/versions/b7006cbdd923_.py

```diff
@@ -0,0 +1,54 @@
+"""empty message
+
+Revision ID: b7006cbdd923
+Revises: 32c1ae6dd23a
+Create Date: 2020-01-26 22:09:30.227267
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = 'b7006cbdd923'
+down_revision = '32c1ae6dd23a'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_unique_constraint('org_email_unique', 'app_user', ['email', 'organisation_id'])
+    op.drop_constraint(u'app_user_email_key', 'app_user', type_='unique')
+    op.add_column('event', sa.Column('application_close', sa.DateTime(), nullable=False))
+    op.add_column('event', sa.Column('application_open', sa.DateTime(), nullable=False))
+    op.add_column('event', sa.Column('offer_close', sa.DateTime(), nullable=False))
+    op.add_column('event', sa.Column('offer_open', sa.DateTime(), nullable=False))
+    op.add_column('event', sa.Column('registration_close', sa.DateTime(), nullable=False))
+    op.add_column('event', sa.Column('registration_open', sa.DateTime(), nullable=False))
+    op.add_column('event', sa.Column('review_close', sa.DateTime(), nullable=False))
+    op.add_column('event', sa.Column('review_open', sa.DateTime(), nullable=False))
+    op.add_column('event', sa.Column('selection_close', sa.DateTime(), nullable=False))
+    op.add_column('event', sa.Column('selection_open', sa.DateTime(), nullable=False))
+    op.alter_column('organisation', 'system_name',
+               existing_type=sa.VARCHAR(length=50),
+               nullable=False)
```

Comment:
This part doesn't look quite right. I think your local database might be slightly out of date. I think when these scripts are generated, it compares what your database looks like currently and what the Python models look like and then does the diff. So I think when you pulled the latest changes, and ran the migration, it added this extra field. In the future, I think make sure to run upgrades whenever you pull latest changes, then run your migration, then the upgrade again. This can get tricky if you've already applied your changes which means you'll first have to downgrade them, delete the script, upgrade, migrate, then upgrade again. But don't worry about this for now. I'll note this down as something to explain in the docs like you mentioned in another comment. 

For now just backspace this line